win32/src/winapi/kernel32/init.rs

```diff
@@ -303,7 +303,7 @@ impl State {
 
     pub fn get_process_heap<'a>(&'a mut self, memory: &mut MemImpl) -> &mut Heap {
         if self.process_heap == 0 {
-            let size = 16 << 20;
+            let size = 24 << 20;
```

Comment:
My eventual idea here (not that you need to fix it, just FYI) is that we could implement heaps as multiple not necessarily contiguous regions.  Like once we fill the first Xmb we just allocate a new block of Xmb and start putting new allocations there.  Kinda gross though.

Reply:
A better heap implementation is going to be necessary soon, these compilers sometimes need more than 24mb of RAM :slightly_smiling_face: